ratelimits/limit.go

```diff
@@ -87,7 +87,12 @@ func loadDefaults(path string) (limits, error) {
 type overrideYAML struct {
 	limit `yaml:",inline"`
 	// Ids is a list of ids that this override applies to.
-	Ids []string
+	Ids []struct {
+		Id string `yaml:"id"`
+		// Comment is an optional field that can be used to provide additional
+		// context for the override.
+		Comment string `yaml:"comment,omitempty"`
```

Comment:
This comment field doesn't appear to actually used by boulder, but that's fine.

Reply:
Yep, but thanks to strict unmarshaling it must be present in the struct.